packages/developer-portal/src/reducers/current-member.ts

```diff
@@ -0,0 +1,45 @@
+import { Action } from '../types/core'
+import { isType } from '../utils/actions'
+import { fetchCurrentMember, fetchCurrentMemberSuccess, updateCurrentMember } from '@/actions/current-member'
+import { MemberModel } from '@reapit/foundations-ts-definitions'
+
+export type CurrentMemberRootState = {
+  data: MemberModel | null
+  isLoading: boolean
+}
+
+export const defaultState: CurrentMemberRootState = {
+  isLoading: false,
```

Comment:
I think should add one more loading for updating
```
update: {
  isLoading: true
}
```
So in this case this update will loading the button only